lib/Wikia/src/Domain/User/Preferences/UserPreferences.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Wikia\Domain\User\Preferences;
+
+class UserPreferences {
+
+	/** @var GlobalPreference[] */
+	private $globalPreferences;
+
+	/** @var array[wikiId => [name => LocalPreference]] */
+	private $localPreferences;
+
+	function __construct() {
+		$this->globalPreferences = [];
+		$this->localPreferences = [];
+	}
+
+	public function setGlobalPreference($name, $value) {
+		$this->globalPreferences[$name] = new GlobalPreference($name, $value);
+		return $this;
+	}
+
+	public function setLocalPreference($name, $wikiId, $value) {
+		$this->localPreferences[$wikiId][$name] = new LocalPreference($name, $value, $wikiId);
```

Comment:
Do we need to assign it to key based on `wikiId`, is there any use-case when there will be loaded local preferences for multiple wikis ??